src/Illuminate/Database/Console/PruneCommand.php

```diff
@@ -157,9 +157,9 @@ protected function models()
     protected function getPath()
     {
         if (! empty($path = $this->option('path'))) {
-            return (new Collection($path))->map(function ($path) {
-                return base_path($path);
-            })->all();
+            return (new Collection($path))
+                ->map(fn ($path) => base_path($path))
```

Comment:
This can be just
```suggestion
                ->map('base_path')
```
or
```suggestion
                ->map(base_path(...))
```